config/cruisecontrol.properties

```diff
@@ -198,6 +198,52 @@ num.concurrent.leader.movements=1000
 execution.progress.check.interval.ms=10000
 
 
+# Configurations for persistent data
+# =======================================
+
+# The method to use to store persisted data. This is the first "persisted.data" config to set which
+# will determine which other configs of the series should be configured. The available options are:
+#   "kafka",
+#   "memory"
+# The default is "memory", which doesn't durably persist any runtime data.
+#persisted.data.persist.method=kafka
+
+# The name of the kafka topic to use to persist data when "persisted.data.persist.method" is set to
+# "kafka". If the topic is not present, then it will be created.
+#persisted.data.kafka.topic.name=__CruiseControlPersistentData
+
+# The number of partitions to ensure are present for the kafka topic. Only applies when
+# "persisted.data.persist.method" is set to "kafka". If the topic has fewer than this number of
+# partitions, then partitions will be added.
+#persisted.data.kafka.topic.partition.count=2
+
+# The replication factor to use for the kafka topic. Only applies when
+# "persisted.data.persist.method" is set to "kafka". Multiple partition replicas are desirable to
+# ensure the topic is reasonably available.
+#persisted.data.kafka.topic.replication.factor=2
+
+# The configs to apply to the kafka topic used to persist Cruise Control data. Only applies if
+# "persisted.data.persist.method" is set to "kafka". This "list" should be a semicolon separated
+# string of 'key=value' pairs. The keys and values need to be valid Kafka Topic configs.
```

Comment:
Ok. I can switch the semicolons to commas and provide an example that demonstrates tidy formatting. In case I've misunderstood your request, the reason for the separator character is that I want to provide a mechanism for users to add any number of key=value pairs to the list rather than mandating which configs are permitted.

Reply:
Sorry I can't switch the semicolons to commas because individual values can have commas. I can make sure the docs make it clear that readable formatting is easy and desirable.